### DISCUSSION TITLE
Query staked balances

### DISCUSSION THREAD

**Discussion by Unknownstarr** [aptos-labs/aptos-developer-discussions#540] [Questions]
### Describe your question in detail.

Hi there - is there an easy way to query staked balances via gsheet ? Can someone also provide some working Aptos RPCs ? thanks !

### Which operating system are you using?

N/A

### Which SDK or tool are you using? (if any)

TypeScript SDK

### Describe your environment or tooling in detail

Trying to drop staked balances on a google sheet.

**Comment by ying-w** [comment]
You can't do it via gsheet but you can get current staking balance using graphql https://cloud.hasura.io/public/graphiql?endpoint=https://api.mainnet.aptoslabs.com/v1/graphql using `current_delegated_staking_pool_balances`

**Comment by Unknownstarr** [comment]
Thanks ! I was able to do it via a script on Google Sheet and Aptosscan.
